auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -1012,7 +1046,7 @@ private LoadedIdentity parseIdentityContents(final XMLStreamReader streamReader,
             }
             List<Credential> credentials = new ArrayList<>();
             do {
-                if (! version.getNamespace().equals(streamReader.getNamespaceURI())) {
+                if ( !(version.getNamespace().equals(streamReader.getNamespaceURI())) ) {
```

Comment:
Looks like an extra set of brackets has been added here.

Reply:
removed